packages/jest-runtime/src/index.js

```diff
@@ -830,6 +830,11 @@ class Runtime {
       return jestObject;
     };
 
+    const retryTimes = (numTestRetries: number) => {
+      this._environment.global.retryTimes = numTestRetries;
```

Comment:
can we put it on the `jest` object instead of as a global?

Reply:
I'll see if that will work

Reply:
@SimenB so I don't think that's going to work. From the context of jest-circus/run.js I don't have access to the jest object. Even if I did I think I would have to add both a getter and a setter for that to work. What do you think?

Reply:
a clean (and private) way of speaking across that boundary would be awesome, but no need to tackle it at this moment 🙂

Reply:
last time i did it with a `Symbol` that is attached to `global`. which is pretty much the same thing, but you only know it's there if you know what you're looking for :)

Reply:
could we do that? Polluting the global namespace in user code seems icky